sensu/standalone_test.go

```diff
@@ -0,0 +1,116 @@
+package sensu
+
+import (
+	"testing"
+
+	stdCheck "github.com/upfluence/sensu-client-go/Godeps/_workspace/src/github.com/upfluence/sensu-go/sensu/check"
+	stdClient "github.com/upfluence/sensu-client-go/Godeps/_workspace/src/github.com/upfluence/sensu-go/sensu/client"
+)
+
+func TestMissingCommandKey(t *testing.T) {
+	standaloneProcessor := &Standalone{check: &stdCheck.Check{}}
+
+	err := standaloneProcessor.execute()
+
+	if err != commandKeyError {
+		t.Fatalf("Expected error to be \"%s\" but got \"%s\" instead!",
```

Comment:
Ups, thanks for spotting those! I think I got them all now.

Are you aware of any linter for Go which might be able to detect such inconsistencies? `go vet` and `golint` don't seem to fit the bill for such a job (out of the box at least) and I seem to be heavily biased from badly formatted C++ codebases...